langchain4j/src/main/java/dev/langchain4j/data/document/DocumentType.java

```diff
@@ -4,5 +4,11 @@ public enum DocumentType {
 
     TXT,
     PDF,
-    HTML
+    HTML,
+    PPT,
+    PPTX,
```

Comment:
I am wondering, do we need to distinguish between PPT and PPTX (same for DOC/DOCX and XLS/XLSX) formats?
For html/htm/xhtml we only have one type: HTML.

Reply:
Oh! I was mistaken. I kept only the latest file extensions!

Reply:
@oognuyh I see that you have removed tests for .doc .ppt .xls files, but why? These file extensions should be supported.
Sorry for miscommunication, what I meant is:
lets have only 3 types of documents in enum: DOC (for .doc and .docx), PPT (for .ppt and .pptx) and XLS (for .xls and .xlsx). Thanks!